code/game/machinery/requests_console.dm

```diff
@@ -51,21 +52,23 @@ GLOBAL_LIST_EMPTY(allRequestConsoles)
 	var/announcementConsole = FALSE
 		// FALSE = This console cannot be used to send department announcements
 		// TRUE = This console can send department announcementsf
-	var/announceAuth = 0 //Will be set to 1 when you authenticate yourself for announcements
-	var/msgVerified = "" //Will contain the name of the person who varified it
-	var/msgStamped = "" //If a message is stamped, this will contain the stamp name
+	var/announceAuth = FALSE //Will be set to TRUE when you authenticate yourself for announcements
+	var/secondaryGoalAuth = FALSE //Will be set to TRUE when you authenticate yourself for requesting a secondary goal
+	var/msgVerified = "Not verified" //Will contain the name of the person who varified it
+	var/msgStamped = "Not stamped" //If a message is stamped, this will contain the stamp name
```

Comment:
```suggestion
	/// Will be set to TRUE when you authenticate yourself for announcements
	var/announceAuth = FALSE
	/// Will be set to TRUE when you authenticate yourself for requesting a secondary goal
	var/secondaryGoalAuth = FALSE
	/// Will contain the name of the person who verified it
	var/msgVerified = "Not verified"
	/// If a message is stamped, this will contain the stamp name
	var/msgStamped = "Not stamped"
```